From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reto Zingg <reto.zingg@jollamobile.com>
Date: Mon, 30 Sep 2013 18:33:20 +0300
Subject: [PATCH] enable systemd activation

apply patch enable-systemd-activation.patch

[maliit] enable systemd activation
---
 connection/org.maliit.server.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/connection/org.maliit.server.service.in b/connection/org.maliit.server.service.in
index 6a3e9216571e97b8319b90bdd86bb14b0c5027b4..f387838dcd59267e34190920a07b97951c8041f1 100644
--- a/connection/org.maliit.server.service.in
+++ b/connection/org.maliit.server.service.in
@@ -1,4 +1,5 @@
 [D-BUS Service]
 Name=org.maliit.server
 Exec=@CMAKE_INSTALL_PREFIX@/bin/maliit-server @MALIIT_SERVER_ARGUMENTS@
+SystemdService=maliit-server.service
 
